pkgs/by-name/mc/mcdreforged/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  fetchFromGitHub,
+  python312Packages,
```

```suggestion
  python3Packages,
```

https://github.com/Fallen-Breath/MCDReforged/blob/850e28fce8ad624fbdd9501f5a5faae3d3ef57f3/setup.py#L23

Does it only support python 3.12 or above elsewhere?

---

there’s no `typing-extensions` in `python3Packages` yet
I find it in `python311Packages` and `python312Packages`

---

There is. How long has it been since you updated your local nixpkgs checkout 🙃

---

Ah…yeah
Actually yesterday…